client/components/App.js

```diff
@@ -1,51 +1,67 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import Cookies from 'js-cookie';
 import Menu from './Menu';
 import Login from './Login';
 import ProductSingle from './ProductSingle';
 import Nav from './Nav';
 import Signup from './Signup';
 import Home from './Home';
 import Order from './Order';
-import Checkout from './Checkout';
```

Comment:
shouldn't we keep these imports?:
`
import Checkout from './Checkout';
import AdminProducts from './AdminProducts';
`

Reply:
Yes, something happened during merge that messed things up.  Fixing ...